### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by Kingno3l**
### Built the desktop version of my portfolio project

- [ ] Used media queries in my CSS file. 

- [ ] Desktop version of template 1 (Screen size of **Desktop: 768px or wider**).

- [ ] In order to lay out the elements on the page, I used Grid and Flexbox.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -57,69 +57,73 @@ <h1 class="section_head">
...
+                        <ul class="frame2">
+                            <li class="tag">html </li>
+                            <li class="tag">css</li>
+                            <li class="tag">javaScript</li>
+                        </ul>
+
+                        <div>
+                            <div class="action">
+                                <button class="buttonaction">See Project</button>
+                            </div>
```
Comment (roshan-bajgain) [inline index.html:97]: - [ ] According to the project requirement you should stick to the design as much as possible like color, Font, size, and spacing. Please adjust the spacing of the cards and also adjust the spacing between the first section and the cards. please see the image below for reference
- [ ] Also use the same color for `canopy`, as it is totally different from the Figma design.
- [ ] You have the same issue with all the cards. So I would suggest please fix the issue for all cards I mention above.
### Your design
your

### Figma design

figma

File: index.html [hunk 2]
```diff
@@ -234,6 +241,7 @@ <h1 class="aboutheader">
                     <button class="buttonaction">Get my resume</button>
                 </div>
             </div>
+            </main>
```
Comment (roshan-bajgain) [inline index.html]: - [ ] You should stick to the design as much as possible like color, Font, and background. Please use the same color for `h3` tag. Please see the image below for reference
- [ ] Also remove the border above the `Framework` to stick to the design
### You design
your

### Figma design
fimga

File: index.html [hunk 3]
```diff
@@ -283,7 +291,8 @@ <h1>Contact me</h1>
                 <input type="text" name="name" placeholder="Enter name" maxlength="30" required>
                 <input type="email" name="email" placeholder="Enter email" required>
                 <textarea name="message" placeholder="Write your message here" maxlength="500" required></textarea>
-                <button type="submit" class="buttonaction">Get In Touch</button>
+                <div class="btnn">
+                <button type="submit" class="buttonaction">Get In Touch</button></div>
             </form>
```
Comment (roshan-bajgain) [inline index.html]: - [ ] Everything is good, there are small changes you need to make to the contact form please implement it. In your contact form, the small background is missing please implement it.
- [ ] I would suggest please decrease the width of form to match the design
### Your design
your

### Fimga design
figaaaaaaaaaaaa

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @kingnoel, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Required Changes ♻️
- [ ] I cannot leave comments there so I am leaving here. Please use the same color for menu icons so it match the design
### Your menu icons
your
- [ ] There is the same issue with the mobile version which is spacing. Please manage the spacing of the mobile version as well.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (Kingno3l) [reply index.html]: Please review the new request before I submit it again.
We have **Limited Request review...**

File: index.html [hunk 4]
```diff
@@ -128,112 +154,119 @@ <h1 class="cardonept">
...
 
 
             </section>
 
-            <section class="cardone">
-                <img src="img/Snapshoot Portfolio-2.png" alt="">
+            <section class="cardone inverse">
+                <div class="imagefit">
+                    <img class="mobile" src="img/Snapshoot Portfolio-2.png" alt="">
+                    <img class="desktop" src="img/Snapshoot Portfolio 4.png" alt="">
```
Comment (Favourezeugwa) [inline index.html:201]: [OPTIONAL] 👍🏽 

- If you have the time, I'd like to kindly suggest that you increase the width and height of the images in your last two project cards. They are a bit too small. 

Here is what I mean 👇🏽 


| Your Project card image  |
| :---:   | 
|small  |

File: index.html [hunk 5]
```diff
@@ -23,13 +23,20 @@
...
+                            <li>About</li>
+                            <li>Contact</li>
+                        </ul>
                     </div>
                 </nav>
             </header>
             <section class="headclass">
                 <h1 class="section_head">
-                    I’m Mariane Glad to see you!
+                    I’m Mariane <br> Glad to see you!
```
Comment (Favourezeugwa) [inline index.html:39]: [OPTIONAL] 👍🏽 

- I am not sure I am reviewing Mariane's Pull Request, Can we change this to KingNoel ?  🌸

**Review by Favourezeugwa — APPROVED** [review]
## STATUS: APPROVED 🥇 🏆 🍹 

Hi @kingnoel , 

I am Favour 👩🏽 , your reviewer for this Pull Request.

Great job implementing all the required changes by the previous code reviewer. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Yayyy! 💃🏽, Congratulations! 🎉 🎉🎉

happy

## Highlights 💯 
- No Linter errors ✔️
- Great design/ UI ✔️
- Good commit messages ✔️
- Descriptive PR ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**